workspaces-tests/src/bin/simple_multisig.rs

```diff
@@ -0,0 +1,126 @@
+#![allow(missing_docs)]
+
+// Ignore
+pub fn main() {}
+
+use std::fmt::Display;
+
+use near_contract_tools::{
+    approval::{
+        self,
+        simple_multisig::{AccountAuthorizer, ApprovalState, Configuration},
+        ApprovalManager,
+    },
+    rbac::Rbac,
+    slot::Slot,
+    Rbac,
+};
+use near_sdk::{
+    borsh::{self, BorshDeserialize, BorshSerialize},
+    env, near_bindgen, BorshStorageKey, PanicOnDefault,
+};
+
+#[derive(BorshSerialize, BorshStorageKey)]
+enum StorageKey {
+    SimpleMultisig,
+}
+
+#[derive(BorshSerialize, BorshDeserialize)]
+enum MyAction {
+    SayHello,
+    SayGoodbye,
+}
+
+impl approval::Action for MyAction {
+    type Output = &'static str;
+
+    fn execute(self) -> Self::Output {
+        match self {
+            Self::SayHello => "hello",
+            Self::SayGoodbye => "goodbye",
+        }
+    }
+}
+
+#[derive(BorshSerialize, BorshStorageKey)]
+enum Role {
+    Multisig,
```

Comment:
@encody My guess is that "Multisig" isn't an example of a typical role name. What could we put here that would be more likely to be used? I think then it will feel more instructive / easier to understand.

Reply:
Multisig seems like a pretty good name to me, and would probably be used (either that or "Member") if the author were implementing a simple multisig wallet, which is what this test contract does.

Did the name hinder you from understanding what the example contract does? If so, it definitely needs to be changed.